### PR TITLE
Gpl 323 view samples plates

### LINKED ISSUE #291: GPL-323 - As a user I want to be able to view ONT samples

### PR DESCRIPTION
**Pull request by harrietc52**
Closes #291

### CODE REVIEW

**Comment by harrietc52** [comment]
TODO: all apollo tests/ mocks

File: apollo.config.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// apollo.config.js
+module.exports = {
+  client: {
+    service: {
+      name: 'my-app',
+      // URL to the GraphQL API
+      url: 'http://localhost:3000/graphql',
```
Comment (sdjm-tessella) [inline apollo.config.js:7]: I thought this was `/v2` rather than `/graphql`?

File: src/main.js [hunk 2]
```diff
@@ -7,13 +7,27 @@ import App from './App.vue'
...
 
 Vue.config.productionTip = false
 Vue.use(BootstrapVue)
 Vue.use(Vuex)
+Vue.use(VueApollo)
+
+
+const apolloClient = new ApolloClient({
+  // You should use an absolute URL here
+  uri: 'http://localhost:3100/v2'
```
Comment (sdjm-tessella) [inline src/main.js:21]: Looks like this overrides the value in the other config file.  Should it be based on the config file or specified here?  This appears to be the correct URL here.

**Comment by harrietc52** [comment]
**On deployment**
Add VUE_APP_TRACTION_GRAPHQL_URL=REPLACE_VUE_APP_TRACTION_GRAPHQL_URL
to TractionUI config

**Comment by sdjm-tessella** [comment]
All looks good to me now

**Pull request merged**